hooks/post-up

```diff
@@ -8,10 +8,16 @@ else
   curl -fLo "$HOME"/.vim/autoload/plug.vim --create-dirs \
       https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
 fi
-vim -u "$HOME"/.vimrc.bundles +PlugUpdate +PlugClean! +qa
+vm -u "$HOME"/.vimrc.bundles +PlugUpdate +PlugClean! +qa
```

Comment:
What happened here?

Reply:
Uhhh, I don't have that change locally. Wild. I just re-pushed my commit and had to force push. Looks like it's not there now?

Reply:
Computers are such a great idea.